docker_tests/test_prod_image.py

```diff
@@ -68,7 +68,7 @@ def test_airflow_command(self, default_docker_image):
     def test_airflow_version(self, default_docker_image):
         """Checking 'airflow version' command. It should return zero exit code."""
         output = run_airflow_cmd_in_docker(["version"], image=default_docker_image)
-        assert "2." in output
+        assert "2." in output or "3." in output
```

Comment:
```suggestion
        assert "3." in output
```

Reply:
Yes. We can get rid of it in main - as long as we don't cherry-pick it it v2-10-test

Reply:
(we are generally using v2-X-stable branch tip to generate the images (once we fix the runners)